### DISCUSSION TITLE
check command output

### DISCUSSION THREAD

**Discussion by liggett2820** [pcdshub/atef#108] [Ideas]
I thought it would be a good idea to incorporate the checklist name along with the PV name when an error displays so that it's easier to find and double check the error in the las window (unless everyone at LCLS has ALL of the PV names memorized. I'm new so I do not have them all memorized).
<img width="499" alt="Screen Shot 2022-07-12 at 12 25 39 PM" src="https://user-images.githubusercontent.com/108372247/178578305-b57c62a5-ee77-443d-8662-ee4fc05f11a6.png">

**Comment by klauer** [comment]
Can you show sample output from `atef check` that you'd like improved?
Working with the sample included in atef, I see PV names like this when disconnected:

```
❌ Error pressures (Make sure we're at vacuum)
├── ❌ Error: Unable to connect to 'AT2L0:SOLID:GCC:01:PRESS_RBV' (pressures) for comparison < 1e-06
├── ❌ Error: Unable to connect to 'AT2L0:SOLID:PIN:01:PRESS_RBV' (pressures) for comparison < 1e-06
├── ❌ Error: Unable to connect to 'BT2L0:PLEG:GCC:01:PRESS_RBV' (pressures) for comparison < 1e-06
...
```

Or this on comparison failures:
```
❌ Error state settings (Check state settings)
├── ❌ Error: AT2L0:XTES:MMS:19:STATE:10:VELO Make sure we have some level of vacuum everywhere (< -1): value of 0.0
└── ❌ Error: AT2L0:XTES:MMS:19:STATE:10:ACCL Make sure we have some level of vacuum everywhere (< -1): value of 0.0
```

Successful ones require `atef check -v` to show the PV names though (verbose mode):
```
✔ Success state settings (Check state settings)
├── ✔ Success: AT2L0:XTES:MMS:19:STATE:10:VELO < 1
└── ✔ Success: AT2L0:XTES:MMS:19:STATE:10:ACCL < 1
```

**Reply by liggett2820** [reply]
yeah. for example, I was test running a passive checkout that I started for Isil to see what errors come up

<img width="792" alt="Screen Shot 2022-07-12 at 2 55 41 PM" src="https://user-images.githubusercontent.com/108372247/178602855-9076f8e4-aaad-45fb-9299-2d8f0357d227.png">

I thought it would be helpful in this case to provide the specific checklist that the user made that corresponds to each failed check along with the PV name. It currently has the PV config name with it, which is helpful, but more information into narrowing down the user inputed check would be more user friendly and efficient (since right now I have 1 PV comparison per checklist).

<img width="498" alt="Screen Shot 2022-07-12 at 3 06 43 PM" src="https://user-images.githubusercontent.com/108372247/178603706-f02159a0-e30a-4f56-857b-4a0d372fdaba.png">

**Reply by ZLLentz** [reply]
So what you're saying is: the "RF Locking" top-level name is included, but the "Diode Power" is missing- as in, we've lost one level of the tree in the output here

**Reply by liggett2820** [reply]
Exactly

**Reply by klauer** [reply]
I misunderstood thinking that the PV name was mistakenly missing - leading to the above query. 

I'll work on adjusting the `check` output to have another level to the tree.

**Comment by klauer** [comment]
Addressed in #110 
Locking conversation because there's no equivalent to "archive" or "close"; requests for further changes should go in an issue
